substrate/client/src/error.rs

```diff
@@ -70,6 +70,12 @@ error_chain! {
 			display("Current state of blockchain has invalid authority count value"),
 		}
 
+		/// Invalid state data.
+		VersionInvalid {
```

Comment:
Should just assume it's the PoC-1 runtime in this case for now, once we ditch the PoC-1 chain that default can be removed.